lib/LemmaBucket.thy

```diff
@@ -100,10 +100,10 @@ lemmas mapM_x_accumulate_checks
 lemma isRight_rel_sum_comb2:
   "\<lbrakk> (f \<oplus> r) v v'; isRight v' \<rbrakk>
        \<Longrightarrow> isRight v \<and> r (theRight v) (theRight v')"
-  by (clarsimp simp: isRight_def)
+  by (cases v; clarsimp)
```

Comment:
wouldn't isRight_def still work here?

Reply:
I think now it might. I fixed up that one before I got the idea for `isRight_def`.

Reply:
(It's the more stable proof, so I don't think we should change it back)